### PR TITLE
docs: update for CNI package change

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

/kind documentation
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:
fixes https://github.com/cri-o/cri-o/issues/2885
(or finishes doing so)

#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -83,6 +83,12 @@ sudo dnf install cri-o
 
 For Fedora, we only support setting minor versions. i.e: `VERSION=1.18`, and do not support pinning patch versions: `VERSION=1.18.3`
 
+Note: as of 1.24.0, the `cri-o` package no longer depends on `containernetworking-plugins` package. This is to allow users to install
```
Comment (TomSweeneyRedHat) [inline install.md:86]:
```suggestion
Note: as of 1.24.0, the `cri-o` package no longer depends on the `containernetworking-plugins` package. Removing this dependency allows users to install
```

File: install.md [hunk 2]
```diff
@@ -101,6 +107,12 @@ curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo
 yum install cri-o
 ```
 
+Note: as of 1.24.0, the `cri-o` package no longer depends on `containernetworking-plugins` package. This is to allow users to install
```
Comment (TomSweeneyRedHat) [inline install.md:110]: If you use prior, do similar here.

File: install.md [hunk 3]
```diff
@@ -153,6 +165,12 @@ runtime_root = "/run/runc"
 
 to `/etc/crio/crio.conf.d/`
 
+Note: as of 1.24.0, the `cri-o` package no longer depends on `containernetworking-plugins` package. This is to allow users to install
```
Comment (TomSweeneyRedHat) [inline install.md:168]: ditto prior

**Review by saschagrunert — APPROVED** [review]
LGTM beside @TomSweeneyRedHat suggestions.

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**